a2t-whisper/docker/cpu/Dockerfile

```diff
@@ -1,31 +1,17 @@
 FROM python:3.10-slim-bullseye
-
-ARG MODEL_ID
-
 WORKDIR /usr/src/app/
 
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
-
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-    build-essential \
-    git \
-    wget \
-    libatlas-base-dev \
-    ffmpeg
-
-COPY requirements.txt /usr/src/app/
-
-RUN pip3 install --no-cache-dir -r requirements.txt
+RUN apt update -qq && apt install -yqq --no-install-recommends \
+    build-essential git wget libatlas-base-dev ffmpeg \
```

Comment:
can someone double-check all the `apt` installs in a ll the `Dockerfile`s? I suspect some of these deps aren't actually required (or maybe some are missing) - e.g. `libatlas-base-dev`?

Reply:
Let's create an issue for that :)